app/uk/gov/hmrc/gform/views/form/snippets/print_pdf_header.scala.html

```diff
@@ -0,0 +1,31 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.gform.sharedmodel.formtemplate.FormTemplate
+@import uk.gov.hmrc.gform.sharedmodel.formtemplate.OfstedDefault
+
+@(
+formTemplate: FormTemplate,
+header: Html
+)(implicit
+messages: Messages)
+
+<div>
+  <span>
+    @if(formTemplate.formCategory.isHmrc) { @messages("branding.hmrc") }
+  </span>
+    @header
+</div>
```

Comment:
Same as before. We already have such a template, it is called `notification_pdf_header.scala.html`